tensorflow/contrib/seastar/seastar_server.h

```diff
@@ -0,0 +1,56 @@
+#ifndef TENSORFLOW_CONTRIB_SEASTAR_SEASTAR_SERVER_H_
+#define TENSORFLOW_CONTRIB_SEASTAR_SEASTAR_SERVER_H_
+#include "core/channel.hh"
```

Comment:
Same above.

Reply:
it's a seastar file, i'll wrapper all these by a facade include file.